### ISSUE TITLE
P0 - Cloud Debugger not enabled by default

### ISSUE THREAD

**Issue by lesv** [GoogleCloudPlatform/tomcat-runtime#113]
@meltsufin 
Looking at how the Dockerfile you appear to do the right thing and inherit from OpenJDK8, but somehow when you start tomcat, it doesn't include all the right JAVAOPTS.

**Comment by ChengyuanZhao** [comment]
looking into it

**Comment by meltsufin** [comment]
It looks like the agent is added correctly, but we're not setting the `CDBG_APP_WEB_INF_DIR` env var for Tomcat. See analogous issue in Jetty: https://github.com/GoogleCloudPlatform/jetty-runtime/issues/174.

**Comment by meltsufin** [comment]
Another issue is that the WAR is unpacked too late. In Jetty, we unpack the war before starting the JVM. See [jetty-50.bash](https://github.com/GoogleCloudPlatform/jetty-runtime/blob/master/jetty9/src/main/docker/50-jetty.bash#L9). A similar unpacking should happen in `50-tomcat.bash` so that the Debugger Agent can find the app class files when it starts.
